conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/MappingRuleInterpreter.scala

```diff
@@ -126,6 +117,30 @@ case class MappingRuleInterpreter(rule: MappingConformanceRule, conformance: Con
     res2
   }
 
+  private def getDefaultValue(mappingTableDef: MappingTable)
+                             (implicit spark: SparkSession, dao: EnceladusDAO): Option[String] = {
+    val defaultMappingValueMap = mappingTableDef.getDefaultMappingValues
+
+    val attributeDefaultValueOpt = defaultMappingValueMap.get(rule.targetAttribute)
+    val genericDefaultValueOpt = defaultMappingValueMap.get("*")
+
+    val defaultValueOpt = attributeDefaultValueOpt match {
+      case Some(_) => attributeDefaultValueOpt
+      case None => genericDefaultValueOpt
+    }
+
+    if (defaultValueOpt.nonEmpty) {
+      val mappingTableSchema = dao.getSchema(mappingTableDef.schemaName, mappingTableDef.schemaVersion)
+      if (mappingTableSchema != null) {
```

Comment:
I would wrap this in an `Option` and pattern match. I'm a little surprised that `DAO` does not return an `Option`, but that's outside the scope of this task.